src/sentry/testutils/factories.py

```diff
@@ -1050,28 +1051,37 @@ def create_sentry_app_avatar(*args, **kwargs):
 
     @staticmethod
     @assume_test_silo_mode(SiloMode.CONTROL)
-    def create_internal_integration(**kwargs):
+    def create_internal_integration(**kwargs) -> SentryApp:
         args = Factories._sentry_app_kwargs(**kwargs)
         args["verify_install"] = False
         user = args.pop("user", None)
-        app = SentryAppCreator(is_internal=True, **args).run(user=user, request=None)
+        app = SentryAppCreator(is_internal=True, **args).run(
+            user=user, request=None, skip_default_auth_token=True
+        )
         return app
 
     @staticmethod
     @assume_test_silo_mode(SiloMode.CONTROL)
-    def create_internal_integration_token(user, install=None, request=None, org=None, scopes=None):
-        if scopes is None:
-            scopes = []
+    def create_internal_integration_token(
```

Comment:
I originally had this factory method be able to create an internal integration inside it if `install` wasn't passed in.
In hindsight, that was a bad decision b/c it breaks single responsibility and makes it murky when to use this factory.
Now, it cannot create an internal integration, and can also handle creating the token from a passed in `SentryApp` in addition to the original `SentryAppInstallation`